crates/re_space_view_time_series/src/aggregation.rs

```diff
@@ -115,9 +118,7 @@ impl MinMaxAggregator {
             let mut acc_max = points[i + j].clone();
             j += 1;
 
-            while j < window_size
-                && i + j < points.len()
-                && are_aggregatable(&points[i], &points[i + j], window_size)
+            while i + j < points.len() && are_aggregatable(&points[i], &points[i + j], window_size)
```

Comment:
I'd do that same change in the average aggregator if only for consistency

Reply:
Oh yeah, good catch